articles/openshift/oshift-how-disable-weak-cipher-suites-enable-hsts.md

```diff
@@ -0,0 +1,105 @@
+
+### UKCloud for OpenShift v3.x
+
+#### Data plane
+
+In order to provide extended client compatibility, the OpenShift router supports cipher suites from TLS 1.0 - TLS 1.2 by default. Some of these cipher suites (especially older ones) are vulnerable to man-in-the-middle (MITM) attacks.
+
+If exposed secure routes are only being accessed from modern clients (that support TLS 1.2 and above) the weak cipher suites can be disabled by adding the following environment variable to any router `DeploymentConfig`s:
+
+```
+oc set env dc/router ROUTER_CIPHERS=modern -n default
+```
+
+This will trigger a rollout of the `DeploymentConfig`, going forward the new router pods will only accept cipher suites from TLS 1.2.
+
+#### Control plane
+
+By default, the OpenShift control plane (web console) only supports TLS 1.2 ciphers, although these do not provide forward secrecy.
+
+It is possible to restrict the ciphers to only ones that provide forward secrecy, to do so please raise a Service Request to the UKCloud OpenShift Support team via the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the UKCloud Portal.
+
+### UKCloud for OpenShift v4.x
+
+#### Data plane
+
+There are no weak ciphers supported by the OpenShift `IngressController` since by default it supports TLS 1.2 cipher suites with forward secrecy. It is possible to further restrict the ciphers by modifying the default `IngressController` object, specifically the value of `IngressController.spec.tlsSecurityProfile`
+
+```
+oc edit ingresscontroller default -n openshift-ingress-operator
+```
+
+#### Control plane
+
+There are no weak ciphers supported by the OpenShift control plane (API) since by default it supports TLS 1.2 and 1.3 cipher suites with forward secrecy.
+
+## Enable HSTS
+
+HSTS (HTTP Strict Transport Security) prevents MITM attacks such as protocol downgrading by setting a header within HTTPS responses. This header (`Strict-Transport-Security`) is cached by a browser and ensures that future requests to the same host always use HTTPS as opposed to HTTP.
+
+> [!NOTE]
+> It is currently not possible to enable HSTS for the OpenShift v3.x web console or the OpenShift v4.x API server.
+
+> [!WARNING]
+> HSTS should only be enabled on routes which currently have a valid certificate; should the certificate become invalid the route will be inaccessible, due to how browsers' HSTS mechanism works. Particular caution should be exercised if you are using a non-standard hostname (i.e. not ending in `<your-cluster-hostname>.ukcloud.com`) for a route - in this case you should ensure a valid certificate is contained within the route spec for edge and re-encrypt routes or within the container if using the pass-through TLS encryption method.
```

Comment:
The last bit of this (starting  ...route spec for edge...) didn't really make sense to me (as much of any of this stuff actually makes sense to me). Can you maybe reword to make it clearer? I don't understand it enough to make a suggestion :(